### PR TITLE
Merge buttons comp

### PR DESCRIPTION
**Pull request by piyushsh25**
Screenshot (186)

### CODE REVIEW

**Review by MaruthiPothuganti — COMMENTED** [review]
Looks Good to merge

File: components/Button/button.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="button.css">
+    <title>Document</title>
```
Comment (junaidshaikh-js) [inline components/Button/button.html:8]: The title should be something meaningful.

File: components/Button/button.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+</head>
+<body>
+  <h1>Buttons</h1>
+<div class="button primary">Primary Solid</div>
+<div class="button disabled">Primary disabled</div>
+<div class="button outline">Primary outline</div>
+<div class="button link">Primary link</div>
+<h1>Icons with text</h1>
```
Comment (junaidshaikh-js) [inline components/Button/button.html:16]: There should be only one h1 per page, read about heading hierarchy.

**Review by junaidshaikh-js — COMMENTED** [review]
Use code formatter.

**Pull request merged**